Research/ResearchUI/iOS/Views/RSDCountdownDial.swift

```diff
@@ -245,26 +254,31 @@ public final class RSDCountdownDial: RSDProgressIndicator, RSDViewDesignable {
     public override func layoutSubviews() {
         super.layoutSubviews()
         
+        _relayoutSubviews()
+        _updateLayerProperties()
+    }
+    
+    private func _relayoutSubviews() {
```

Comment:
this is only going to redraw the ring and dial if and only if the bounds have changed. Because of this, up above where you are calling `_relayoutSubviews()` in the `didSet` of the `isClockwise` var, it will not actually redraw those layers to clockwise.  Additionally, you need to call `_updateLayerProperties()` *after* recreating the layers in order to properly set the properties on those layers.

Recommend this instead:

```
    /// The default countdown dial progress fills up clockwise starting at 12 o'clock.
     /// If 'isClockwise' is false, the dial will fill up counter-clockwise starting at 12 o'clock.
     @IBInspectable
     public var isClockwise: Bool = true {
         didSet {
             _redrawRings()
             _updateLayerProperties()
         }
     }

    public override func layoutSubviews() {
         super.layoutSubviews()

         let bounds = self.layer.bounds
         if _rect == nil || !bounds.equalTo(_rect!) {
              _rect = bounds
             _redrawRings()
         }
         _updateLayerProperties()
     }

    private func _redrawRings() {
             // Remove existing layers
             ringLayer?.removeFromSuperlayer()
             dialLayer?.removeFromSuperlayer()

             // Create the circle path
             let path = createCirclePath().cgPath

             ringLayer = CAShapeLayer()
             ringLayer.path = path
             layer.insertSublayer(ringLayer, at: 0)
             ringLayer.frame = bounds

             dialLayer = CAShapeLayer()
             dialLayer.path = path
             layer.addSublayer(dialLayer)
             dialLayer.frame = bounds
         }
```